deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf

```diff
@@ -0,0 +1,29 @@
+<match kube-audit.**>
+    @type relabel
+    @label @KUBEAUDIT
+</match>
+<label @KUBEAUDIT>
+    <filter kube-audit.**>
```

Comment:
Indentation looks wrong, please correct.

Reply:
Yep tabs were set to 4 spaces, should be 2 now..